nmostesting/suites/IS0402Test.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, apis, **kwargs):
         self.is04_query_utils = IS04Utils(self.query_url)
         self.test_data = self.load_resource_data()
         self.subscription_data = self.load_subscription_request_data()
+        self.auth = auths[0]
```

Comment:
Should this be like this... ?

```suggestion
        self.auth = auths[0] if auths else None
```

Reply:
Updated